kak-lsp.toml

```diff
@@ -4,20 +4,37 @@ verbosity = 2
 # Semantic tokens support
 # See https://github.com/microsoft/vscode-languageserver-node/blob/8c8981eb4fb6adec27bf1bb5390a0f8f7df2899e/client/src/semanticTokens.proposed.ts#L288
 # for token/modifier types.
+[[semantic_tokens]]
+name = "type"
+face = "type"
 
-[semantic_tokens]
-type = "type"
-variable = "variable"
-namespace = "module"
-function = "function"
-string = "string"
-keyword = "keyword"
-operator = "operator"
-comment = "comment"
-
-[semantic_token_modifiers]
-documentation = "documentation"
-readonly = "default+d"
```

Comment:
There is no 1:1 "catch-all" equivalent.
However, `documentation` will likely only ever appear on a `comment`, so the near-equivalent for that would be

```toml
[[semantic_tokens]]
name = "comment"
face = "documentation"
modifiers = ["documentation"]
```

I guess I'll add that, since it should make for a good example.

For `readonly`, it's still possible to get the same functionality. But it would need a bit more work, since individual entries for every token that can have that modifiers need to be added.